lib/omnibus/fetchers/net_fetcher.rb

```diff
@@ -125,6 +125,11 @@ def checksum
       source[digest_type]
     end
 
+    # Set to false to extract archives into the root source_dir (default).
+    # Set to true to always extract underneath project_dir
+    #
+    attr_accessor :isolate_extraction
```

Comment:
The intent of the flag is to let fetcher know whether it should extract to Config.source_dir or to project_dir since the previous meaning of "relative_path" was to provide some path from Config.source_dir.  Maybe I could call the flag "legacy_behavior" or something like that?